tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+from .utilits import name_generator
+
+
+@pytest.fixture
+def customer_data(last_name="Иванов", post_code="0001252667"):
+    first_name = name_generator(post_code)
+    return {
+        "first_name": first_name,
+        "last_name": last_name,
+        "post_code": post_code,
+    }
+
+
+@pytest.fixture
+def driver():
+    options = Options()
+    options.add_argument('--disable-application-cache')
+    options.add_argument('--headless')
+    options.add_argument("--no-sand box")
+    options.add_argument("--window-size=1920,1080")
```

Comment:
Можно использовать **black** для автоматического форматирования кода.
```suggestion
    options.add_argument("--disable-application-cache")
    options.add_argument("--headless")
    options.add_argument("--no-sand box")
    options.add_argument("--window-size=1920,1080")
```

Reply:
Спасибо. ПОдскажи, стоит его в CI CD включить? или достаочно локально гонять?